src/table.js

```diff
@@ -1381,6 +1381,111 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`
       callback
     );
   }
+
+  /**
+   * Generates Consistency-Token and check consistency for generated token
+   * In-case consistency check returns false, retrial is done in interval
+   * of 5 seconds till 10 minutes, after that it returns false.
+   *
+   * @param {function(?error, ?boolean)} callback The callback function.
+   * @param {?Error} callback.err An error returned while making this request.
+   * @param {?Boolean} callback.resp Boolean value.
+   */
+  waitForReplication(callback) {
+    // handler for generated consistency-token
+    const tokenHandler = (err, resp) => {
+      if (err) {
+        return callback(err);
+      }
+
+      // set timeout for 10 minutes
+      const timeoutAfterTenMinutes = setTimeout(() => {
+        callback(null, false);
+      }, 10 * 60 * 1000);
+
+      // method checks if retrial is required & init retrial with 5 sec delay
+      const retryIfNecessary = (err, res) => {
+        if (err) {
+          clearTimeout(timeoutAfterTenMinutes);
+          return callback(err);
+        }
+
+        if (res === true) {
+          clearTimeout(timeoutAfterTenMinutes);
+          return callback(null, true);
+        }
+
+        setTimeout(launchCheck, 5000);
+      };
+
+      // method to launch token consistency check
+      const launchCheck = () => {
+        const token = resp;
+        this.checkConsistency(token, retryIfNecessary);
+      };
+
+      launchCheck();
+    };
+
+    // generate consistency-token
+    this.generateConsistencyToken(tokenHandler);
+  }
+
+  /**
+   * Generates Consistency-Token
+   * @param {function(?error, ?boolean)} callback The callback function.
+   * @param {?Error} callback.err An error returned while making this request.
+   * @param {?String} callback.consistency_token The generated consistency token.
+   */
+  generateConsistencyToken(callback) {
+    const reqOpts = {
+      name: this.name,
+    };
+
+    this.bigtable.request(
+      {
+        client: 'BigtableTableAdminClient',
+        method: 'generateConsistencyToken',
+        reqOpts: reqOpts,
+      },
+      (err, res) => {
+        if (err) {
```

Comment:
This is just a code style comment that would match how we usually structure the code, so feel free to take it or leave it.

```js
if (err) {
  callback(err);
  return;
}

callback(null, res.consistencyToken);
```

The pattern is simply to handle errors in its own block, and let the success condition remain un-indented.

Reply:
Done https://github.com/googleapis/nodejs-bigtable/pull/82/commits/02debc6039125a03b782c1a1a5216abb35e2ce84